unit_test.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
This file stolen shamelessly from https://gitlab.cee.redhat.com/insights-platform/advisor-backend/-/blob/master/unit_test.sh